python_modules/libraries/dagster-fivetran/dagster_fivetran/translator.py

```diff
@@ -80,7 +117,7 @@ def get_asset_spec(self, props: FivetranConnectorTableProps) -> AssetSpec:
         schema_name, table_name = props.table.split(".")
         schema_entry = next(
             schema
-            for schema in props.schemas["schemas"].values()
+            for schema in props.schema_config["schemas"].values()
```

Comment:
why the name change?

Reply:
To reflect Fivetran's ontology, see [here](https://fivetran.com/docs/rest-api/api-reference/connector-schema/connector-schema-config)